api/users/models.py

```diff
@@ -194,7 +194,10 @@ def full_name(self):  # type: ignore[no-untyped-def]
 
     @property
     def email_domain(self):  # type: ignore[no-untyped-def]
-        return self.email.split("@")[1]
+        email_parts = self.email.split("@")
+        if len(email_parts) > 1:
+            return email_parts[1]
+        return None
```

Comment:
This looks good as functionality **only if** domain-less email addresses exist in our database or use cases. Otherwise, I'd rather prevent invalid email addresses at input validation layer, or perhaps even a database constraint if really necessary.

Reply:
looks better. Overrid the token serializer to validate there but i'd prefer not touch the database as this is not so critical